docs/guide/installation/advanced.rst

```diff
@@ -36,14 +36,18 @@ It is however *strongly* recommended to have all the dependencies installed (wit
 
 There are multiple options depending on how many optional dependencies you want to install:
 
-To install sunpy with ``pip`` including optional dependencies (recommended), simply run::
+To install sunpy with ``pip`` including all optional dependencies (*recommended*), simply run::
```

Comment:
I am wondering if we want to revamp this entire section.

Reply:
How?

Reply:
Like so.